Sources/xcproj/CommentedString.swift

```diff
@@ -19,8 +19,14 @@ struct CommentedString {
         self.comment = comment
     }
 
-    private static let invalidCharacters = CharacterSet.alphanumerics.inverted
-        .subtracting(CharacterSet(charactersIn: "_./"))
+    private static var invalidCharacters: CharacterSet = {
+        var invalidSet = CharacterSet(charactersIn: "_$")
```

Comment:
It's interesting the _ and $ get special treatment. It must be because Xcode uses these extensively in build settings